app/main.py

```diff
@@ -181,11 +188,15 @@ def move():
     # Route setter
     safety = safe(grid, me, data)
     route = path(me, foods[0], grid)
+    routeTail = path(me, tails[0], grid) # Modeled after Dennis' path finding for food, but replaces - hopefully - with tails.
```

Comment:
Good effort. Although I will look to incorporate the routeTail into path. If you look at `path`, you will see how this might not be the best way to do what you are suggesting!